motorconf/motorconf.coffee

```diff
@@ -43,9 +49,16 @@ program
 		console.log '   <operation> : use "write" to write a new parameter value in flash,'
 		console.log '                  and "read" to get the current parameter value'
 		console.log '   <parameter> : a parameter name among : '
-		console.log '                  wheelsgap, tickspermeter, linacc, angacc, linspeed, angspeed,'
-		console.log '                  linp, lini, lind, angp, angi, angd, ldir, lwheeldir, lwheeloffset,'
-		console.log '                  rdir, rwheeldir, rwheeloffset'
+		console.log '       wheelsgap         : distance between the encoder wheels in mm'
+		console.log '       tickspermeter     : number of ticks when the robot moves 1m forward'
+		console.log '       linacc, angacc    : accelerations during speed up and slow down (mm/s^2 and deg/s^2)'
+		console.log '       linspeed, angspeed: cruise speeds (mm/s and deg/s)'
+		console.log '       linp, lini, lind  : linear PID coefficients'
+		console.log '       angp, angi, angd  : angular PID coefficients'
+		console.log '       ldir, lwheeldir   : left side hardware switches (0 or 1)'
+		console.log '       rdir, rwheeldir   : right side hardware switches (0 or 1)'
+		console.log '       distance          : distance to move (write) or traveled (read) in mm, signed'
+		console.log '       heading           : heading to turn (write) or current heading (read), 0 to 360 CCW'
```

Comment:
Maybe you could precise the order "distance" is relativ, whereas heading is absolute